omni_blocks/__init__.py

```diff
@@ -1,3 +1,3 @@
-__version__ = '2.0.0'
+__version__ = '2.0.3'
```

Comment:
Is there a reason we've bumped 3 patch versions here?

Reply:
To emphasize the fast progress i guess...

Reply:
Looks like it was tagged, but not updated in the `__init__`

Reply:
@dammitjim I was very tired on Friday and made a mess of the two patch versions I tagged previously, could ignore those though and go to 2.0.1 if that makes more sense?

Reply:
I only realised this needed updating once I tried to `pip install` it